src/components/Nav.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Nav() {
+  return (
+    <header>
+      <nav>
+        <ul>
+          <li className="nav-brand">Bookstore</li>
+        </ul>
+        <ul>
+          <li>
+            <Link to="/" className="nav-links">
+              Books
+            </Link>
+          </li>
+          <li>
+            <Link to="/components/Catigory" className="nav-links">
+              Catigory
```

Comment:
- Please, kindly correct your spelling for the category.

Reply:
Thanks for the observations, I will resolve them quickly.